README.md

```diff
@@ -6,9 +6,10 @@ snappy-java is a Java port of the snappy
 ## Features
   * Fast compression/decompression around 200~400MB/sec.
   * Less memory usage. SnappyOutputStream uses only 32KB+ in default.
-  * JNI-based implementation to achieve comparable performance to the native C++ version.  
+  * JNI-based implementation to achieve comparable performance to the native C++ version.
      * Although snappy-java uses JNI, it can be used safely with multiple class loaders (e.g. Tomcat, etc.).
   * Compression/decompression of Java primitive arrays (`float[]`, `double[]`, `int[]`, `short[]`, `long[]`, etc.)
+     * To improve the compression ratios of these arrays, you can use a fast data-rearrangement implementation ([`BitShuffle`](http://static.javadoc.io/org.xerial.snappy/snappy-java/1.1.3-M1/org/xerial/snappy/BitShuffle.html)) before compression
```

Comment:
We are already using sonatype repo for javadoc:  https://oss.sonatype.org/service/local/repositories/releases/archive/org/xerial/snappy/snappy-java/1.1.3-M1/snappy-java-1.1.3-M1-javadoc.jar/!/org/xerial/snappy/BitShuffle.html